style.css

```diff
@@ -0,0 +1,111 @@
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+h1{ 
+  color: #FFF0FF;
+}
```

Comment:
This part should style used the class of the container(I suggest using the flex-box styling this)

Reply:
- [ ] Please do your best to use kind words like `Please` and `Kindly` when asking for changes.